Mvc.NoFun.sln

```diff
@@ -83,24 +81,6 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Microsoft.AspNetCore.Mvc.Te
 EndProject
 Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "MvcSandbox", "samples\MvcSandbox\MvcSandbox.xproj", "{14ED4476-9F24-4776-8417-EA6927F6C9C9}"
```

Comment:
That'd be trickier because KoreBuild would need to know of said new directory. It specifically looks for `src`, `test`, and `samples` when building.